scripts/check_total_coverage.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+coverage_dir=$(PWD)/coverage
```

Comment:
please check previous to this, that the current dir is avalanche-cli repo dir, so PWD uses the correct assumption.

Reply:
(parent of the parent of the script)

Reply:
as my above comment - ut script gave me 'permission denifed' failure without adding this `PWD`. still don't know why.

Reply:
Please check something like

```
if ! [[ "$0" =~ scripts/check_coverage.sh ]]; then
  echo "must be run from repository root"
  exit 1
fi
```

before any other command

Reply:
added in